src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  padding: 32px $hug $hug;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: $background-color;
+
+  transition: all 0.5s ease-in-out;
```

Comment:
![image](https://github.com/mate-academy/layout_catalog/assets/143347271/e106813e-42b6-489f-9a28-7131a6a17cb2)
why 0.5s transition?
And where is transition for other classes?
Also i think you should add cursor: pointer; for card.

Reply:
The 0.5s transition looks smoother to me.
There is no demand for adding `cursor:pointer` on the CHECKLIST